src/main/groovy/pl/allegro/tech/build/axion/release/infrastructure/di/ScmRepositoryFactory.groovy

```diff
@@ -14,14 +14,19 @@ class ScmRepositoryFactory {
 
     private static final String GIT = 'git'
 
+    private static final String REPOSITORY_PUSH_TAGS_ONLY_FLAG = "repository.pushTagsOnly";
```

Comment:
could you change the name to `release.pushTagsOnly`? Name clash is very unlikely for such a long flag, but i would like to keep everything in `release` namespace